### PR TITLE
Feature | New homepage design

### PR DESCRIPTION
**Pull request by monsteronfire**
This PR contains:
1. New animated header image
2. New colour scheme and overall theme
3. Updated footer links and simplified header menu
4. Prominent "hire us" button
5. Updated section content
6. Animated section images
7. A fix for kramdown vulnerability
8. Basically a bunch of stuff

## Before
Screenshot_2020-08-10 Rapid River Software

## After
Screenshot_2020-08-10 Rapid River Software

### CODE REVIEW

File: _includes/shared/footer.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                        </ul>
+                    </div>
+                    <div class="credits">
+                        <p><strong>Credits and Attribution</strong></p>
+                        <ul class="creditlist">
+                            <li>
+                                Illustration Elements by <a href="https://stories.freepik.com/" target="_blank" rel="noopener noreferrer" class="link">Stories by Freepik</a>
+                            </li>
+                            <li>
+                                Background patterns by<a href="https://www.heropatterns.com/" target="_blank" rel="noopener noreferrer" class="link">Hero Patterns</a>
```
Comment (calumhalcrow) [inline _includes/shared/footer.html:23]: Need a space here after "by".

**Comment by monsteronfire** [comment]
@calumhalcrow Re: above feedback

1. Noted, will work on a fix
2. I felt that choosing characters with obviously more "ethnic" hair and facial features was enough to indicate diversity without having to explicitly make the illustrations have dark blue skin. Since the illustrations are monochromatic, I intentionally made them all have zero skin tone to avoid having to struggle with not representing enough groups. We can discuss this further if you feel strongly about giving them skin tones, though.
3. Noted. Will revert to the old menu

File: _includes/shared/footer.html (hunk 1 above)
Comment (monsteronfire) [reply _includes/shared/footer.html:23]: Fixed.

**Comment by monsteronfire** [comment]
@calumhalcrow All the points from your feedback have either been discussed or addressed. ⚡

**Comment by monsteronfire** [comment]
@calumhalcrow Thanks for the reviews! Merging.

**Pull request merged**